macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -29,7 +29,9 @@ open class AvatarView: NSView {
                       contactImage: NSImage? = nil) {
 
 		// Prefer contactEmail to contactName for uniqueness
-		avatarBackgroundColor = AvatarView.backgroundColor(for: AvatarView.colorIndex(for: contactEmail ?? contactName ?? ""))
+		let color = AvatarView.getColor(for: AvatarView.colorIndex(for: contactEmail ?? contactName ?? ""))
+		avatarBackgroundColor = color.background.resolvedColor()
+		initialsFontColor = color.foreground.resolvedColor()
```

Comment:
Note, this doesn't actually fix #392 because this algorithm differs from iOS. if both primaryText (contactName) and secondaryText (contactEmail) are available, then iOS concatenates the strings. Check out this method: https://github.com/microsoft/fluentui-apple/blob/d3126dd5097de7fd3c551e2e038bec3e5232408a/ios/FluentUI/Avatar/InitialsView.swift#L91

Reply:
Let's restrict the scope of this PR to colors and theme, I will add another PR to address the issue.